reference-data/CutOffDateTimeCodeList-CS_v1.0.0.csv

```diff
@@ -1,16 +1,16 @@
-CutOffDateTimeCode,CutOffDateTimeName
-DCO,Documentation cut-off
-VCO,VGM cut-off
-FCO,FCL delivery cut-off
-LCO,LCL delivery cut-off
-PCO,Port cut-off
-ECP,Empty container pick-up date and time
-EFC,Earliest full-container delivery date
-RCO,Reefer cut-off
-DGC,Dangerous Goods cut-off
-OBC,OOG/BB cut-off
-TCO,Transhipment cut-off
-STA,Standard booking acceptance
-SPA,Special booking acceptance
-CUA,Customs Acceptance
-AFC,Advanced filling cut-off
+CutOffDateTimeCode,CutOffDateTimeName,Description
+DCO,Documentation cut-off,Document cut-off time for Shipping Instructions submission.
+VCO,VGM cut-off,Cut-off time for Verified Gross Mass (VGM) submission.
+FCO,FCL delivery cut-off,Latest deadline for the cargo delivery to the port or terminal in order to make the schedule requested. Typically, two days before the expected departure date, but will vary based on the carrier and the port.

```

Comment:
This will not work - if a "comma" (`,`) is included in the description - the parsing will mess up... Please add quotes around descriptions that include comma :-)